source/slang/type-layout.cpp

```diff
@@ -1824,10 +1862,9 @@ static RefPtr<TypeLayout> maybeAdjustLayoutForArrayElementType(
     }
 }
 
-SimpleLayoutInfo GetLayoutImpl(
+static TypeLayoutResult _createTypeLayout(
```

Comment:
Seems like functions returning TypeLayoutResult should have a name perhaps reflecting that to contrast with creating a TypeLayout.

Reply:
Fair point. I hope that the real long-term fix is to replace them with functions that just return `RefPtr<TypeLayout>` instead, since `SimpleLayoutInfo` is on paper a subset of the information stored in a `TypeLayout` (and it is just that extracting that information back out of the representation could be a little bit fiddly).